packages/visualizations/tests/integration/components/number-format-selector-test.js

```diff
@@ -0,0 +1,79 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import wait from 'ember-test-helpers/wait';
+
+let Template = hbs`
+  {{number-format-selector
+    format=format
+    onUpdateFormat=(action onUpdateFormat)
+  }}`;
+
+moduleForComponent('number-format-selector', 'Integration | Component | number format selector', {
+  integration: true,
+  beforeEach() {
+    this.set('format', '$0,0[.]00');
+    this.set('onUpdateFormat', () => null);
+  }
+});
+
+test('updateFormat from radio button', function (assert) {
+  assert.expect(1);
+
+  this.set('onUpdateFormat', result => {
+    assert.equal(result.format,
+      '0,0.00',
+      'onUpdateFormat action is called by radio button');
+  });
+
+  this.render(Template);
+
+  Ember.run(() => {
```

Comment:
Can we import run from `@ember/runloop`?

Reply:
update

Reply:
Did we do this?